### DISCUSSION TITLE
how to really disable receivng deployment notification ?

### DISCUSSION THREAD

**Discussion by applemate** [vercel/community#2095] [Help] [answered]
the successful deployment emails are drive me nuts, gotta delete ton of them.

I tried to disable notification on github, un checked every kind notification, yet still got notifications from vercel deployment.

How to really disable it ?

**Comment by amyegan** [comment, marked as answer]
Hi @craigcosmo. We don't have control over the emails that are sent by GitHub. You'll need to reach out to the [GitHub community](https://github.com/orgs/community/discussions) or [GitHub support](https://support.github.com/request) for help [configuring GitHub notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications). 

You can change your Vercel notification settings at https://vercel.com/account/notifications
